VK/VK/SearchGroupScreen/Cells/CommonGroupTableViewCell.swift

```diff
@@ -32,7 +33,15 @@ final class CommonGroupTableViewCell: UITableViewCell {
     // MARK: - Private methods
 
     private func configreAvatarImageView(avatarImageName: String) {
-        groupAvatarImageView.image = UIImage(named: avatarImageName)
+        DispatchQueue.global().async {
+            guard let url = URL(string: avatarImageName) else { return }
+            let imageData = try? Data(contentsOf: url)
+            let image = UIImage(data: imageData ?? Data())
```

Comment:
```suggestion
            guard let url = URL(string: avatarImageName) else { return }
            let imageData = try? Data(contentsOf: url)
            let image = UIImage(data: imageData ?? Data())
```
кажется это все в один Guard можно засунуть и избавиться от опционалов